src/handle_provide_parameter.c

```diff
@@ -279,14 +279,16 @@ static void handle_claim_airdrop(ethPluginProvideParameter_t *msg, context_t *co
 
 static void handle_governor_cast_vote(ethPluginProvideParameter_t *msg, context_t *context) {
     switch (context->next_param) {
-        case PROPOSAL_ID:  // proposalId
-            copy_parameter(context->lisk.body.governor.proposal_id, msg->parameter, INT256_LENGTH);
+        case ID:  // proposalId
+            copy_parameter(context->lisk.body.governor.data.first,
+                           msg->parameter,
+                           sizeof(context->lisk.body.governor.data.first));
             context->next_param = SUPPORT;
             break;
         case SUPPORT:  // support
-            copy_parameter(context->lisk.body.governor.support,
+            copy_parameter(context->lisk.body.governor.data.secound,
                            msg->parameter,
-                           sizeof(context->lisk.body.governor.support));
+                           sizeof(context->lisk.body.governor.data.secound));
```

Comment:
why key is SUPPORT, but now changed to `data.second`? same for other places.

Reply:
I defined a new struct which is possible to use in some functions instead of having a specific variable name for each function.

The function and the contract names are clear, I just used a new struct to reduce our redundancy.